changelogs/unreleased/209-deprecate-the-net-module.yml

```
@@ -0,0 +1,7 @@
+description: Deprecate the net module.
+issue-nr: 209
+issue-repo: net
+change-type: minor
+destination-branches: [master]
```

The deprecation log message will also show up on iso6, so we need to make sure it's also mentioned in the changelog.
```suggestion
destination-branches: [master, iso6]
```